tgui/packages/tgui/interfaces/common/sorting/SortableTable/type.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author mloccy (https://github.com/mloccy)
+ * @license ISC
+ */
+
+import { BooleanLike } from "common/react";
+import { InfernoNode } from "inferno";
+import { SortDirection } from "../type";
+
+/** Configuration for Sortable Table header row and callback functions */
+export interface SortableTableHeaderConfig {
+
+  /** Presentation of header cell for this field */
+  children: InfernoNode,
```

Comment:
`children` as a non-component name is somewhat surprising to the reader, prefer calling this `headerContents` or something that isn't a React/Inferno built-in concept.